Polygon_mesh_processing/test/Polygon_mesh_processing/remeshing_test.cpp

```diff
@@ -155,25 +155,21 @@ int main(int argc, char* argv[])
 
   std::cout << "done." << std::endl;
 
-  std::set<face_descriptor> patch;
-  std::copy(pre_patch.begin(), pre_patch.end(),
-            std::inserter(patch, patch.begin()));
-
-  std::cout << "Start remeshing of " << selection_file
-    << " (" << patch.size() << " faces)..." << std::endl;
+  //std::cout << "Start remeshing of " << selection_file
+  //  << " (" << patch.size() << " faces)..." << std::endl;
 
   CGAL::Timer t;
-  t.start();
-
-  PMP::isotropic_remeshing(
-    patch,
-    target_edge_length,
-    m,
-    PMP::parameters::number_of_iterations(nb_iter)
-    .protect_constraints(false)
-    );
-  t.stop();
-  std::cout << "Remeshing patch took " << t.time() << std::endl;
+  //t.start();
+
+  //PMP::isotropic_remeshing(
+  //  patch,
+  //  target_edge_length,
+  //  m,
+  //  PMP::parameters::number_of_iterations(nb_iter)
+  //  .protect_constraints(false)
+  //  );
+  //t.stop();
+  //std::cout << "Remeshing patch took " << t.time() << std::endl;
 
```

Comment:
no, it has been commented on purpose, because `patch` is not valid anymore when this line is reached.

However, I would like to keep this code here as a reminder that I would like to compute a new patch, using `detect_sharp_edges` and `connected_component` in a near future